src/po-helpers.js

```diff
@@ -190,7 +190,8 @@ export function hasTranslations(translationObj) {
 export function isFuzzy(translationObj) {
     return (
         translationObj && translationObj.comments
-        && translationObj.comments.flag === 'fuzzy');
+        && translationObj.comments.flag
+        && translationObj.comments.flag.includes('fuzzy'));
```

Comment:
I think this is good enough.
A bit quirky, but I figure that the format of PO files is changed rather rarely.

As a though experiment:

One alternative would be to parse the comment (split on comma, trim whitespace).

Another alternative would be match a more exact pattern.

```suggestion
        && translationObj.comments.flag.includes(', fuzzy'));
```

Ref:
https://github.com/autotools-mirror/gettext/blob/6456980949f3766b366d3a7afba05d7fefe27cd0/gettext-tools/src/read-stringtable.c#L462-L474